thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiFragment.java

```diff
@@ -43,6 +43,11 @@
 
     private static final String SAVED_STATE_PRESENTER_ID = "presenter_id";
 
+    /**
+     * enables debug logging during development
+     */
+    private static final boolean DEBUG = false;
```

Comment:
`DEBUG` is a little bit confused. It sounds like "debug product flavor" or something like that. 
It is more a `ENABLE_LOGGING`, isn't it? 🤔